lib/views/git-timings-view.js

```diff
@@ -423,9 +423,9 @@ export default class GitTimingsView extends React.Component {
     e.preventDefault();
     dialog.showOpenDialog({
       properties: ['openFile'],
-    }, async filenames => {
-      if (!filenames) { return; }
-      const filename = filenames[0];
+    }).then(async ({filePaths}) => {
```

Comment:
And here ☝🏻 

Sorry, should have clarified that I meant at all three callsites on the first review 👀

Reply:
oh ok no issue, i can convert them, i m still here.